StreamingAppDemo/app/src/main/java/com/fortyseven/degrees/streamingapp/home.kt

```diff
@@ -0,0 +1,44 @@
+package com.fortyseven.degrees.streamingapp
+
+import io.reactivex.Observable
+import io.reactivex.schedulers.Schedulers
+
+
+sealed class HomeVM {
```

Comment:
I usually call these "ViewState" to avoid confusion with what people uses to call ViewModels in Android.

Reply:
I'll change it into `HomeViewState`, makes much more sense!